plugins/woocommerce/client/legacy/css/woocommerce-blocktheme.scss

```diff
@@ -0,0 +1,475 @@
+/**
+* woocommerce-blocktheme.scss
+* Block theme default styles to ensure WooCommerce looks better out of the box with block themes that are not optimised for WooCommerce specifically.
+*/
+@import "variables";
+
+/**
+* Layout
+*/
+.woocommerce-cart,
+.woocommerce-account,
+.woocommerce-checkout {
+	.wp-block-post-title,
+	main .woocommerce {
+		// Allow Cart/Checkout/Account pages more space to breathe.
+		max-width: 1000px;
+	}
+}
+
+/**
+* General
+*/
+.woocommerce {
+	mark {
+		// Style the mark element in theme colors.
+		// For details see https://github.com/woocommerce/woocommerce/pull/31631.
+		background-color: var(--wp--preset--color--foreground, $highlight);
+		color: var(--wp--preset--color--background, $highlightext);
+	}
+
+	/**
+	* Buttons
+	*/
+	a.button,
+	button[name="add-to-cart"],
+	input[name="submit"],
+	button.single_add_to_cart_button,
+	button[type="submit"]:not(.wp-block-search__button) {
+		// Slightly increase CTA size across the board.
+		padding: 0.9rem 1.1rem;
+	}
+
+	#respond input#submit.alt,
```

Comment:
We may want to consider broadening coverage here? 🤔  Recommended:

```
	#respond input#submit,
	a.button,
	button.button,
	input.button,
	a.button.alt {}
```

| Before | After |
|---------|------------|
| ![Screen Shot 2022-07-19 at 6 00 59 PM](https://user-images.githubusercontent.com/405912/179856754-05e4a9d6-f38b-4e59-8599-eb9458f0338f.png) | ![Screen Shot 2022-07-19 at 6 02 39 PM](https://user-images.githubusercontent.com/405912/179856796-e4ecd5d6-2048-4fd3-89d8-5b30fbe819d5.png) |
| ![Screen Shot 2022-07-19 at 6 01 15 PM](https://user-images.githubusercontent.com/405912/179856905-bfa43fea-214d-4f18-b251-92fb48a50ca6.png) | ![Screen Shot 2022-07-19 at 6 04 23 PM](https://user-images.githubusercontent.com/405912/179856973-81750762-cea2-439c-bdbd-429d62a33406.png) |
| ![Screen Shot 2022-07-19 at 6 01 47 PM](https://user-images.githubusercontent.com/405912/179857041-16a7f30a-f4e0-4835-b45c-bb3b87cac4ec.png) | ![Screen Shot 2022-07-19 at 6 02 24 PM](https://user-images.githubusercontent.com/405912/179857082-38c5af16-4480-4f69-a202-58c64caed517.png) |

Reply:
Thanks for your suggestion! The selectors `button.button, a.button` with the hover state breaks the current UI experience with TT2:

## Before:

![Screen Capture on 2022-07-20 at 15-00-32](https://user-images.githubusercontent.com/4463174/179988259-c73ac581-2f20-4561-a483-8e47e3dad953.gif)


![Screen Capture on 2022-07-20 at 14-59-57](https://user-images.githubusercontent.com/4463174/179988244-897b7654-12e7-4942-a913-7980c65c244a.gif)


## After (with your suggestions):

![Screen Capture on 2022-07-20 at 14-58-03](https://user-images.githubusercontent.com/4463174/179987887-80a30b8c-dc60-4c66-a942-cd3f8f6cac7b.gif)

![Screen Capture on 2022-07-20 at 14-57-26](https://user-images.githubusercontent.com/4463174/179987899-da58dfb1-c1fb-4da9-8032-9bd4619f9c99.gif)


So, I extract the [two selectors](https://github.com/woocommerce/woocommerce/pull/33518/commits/36b85c13538a1d5911e9904ecc0e5c65d42fbb87#diff-4d8249aa26cad0c3201e91cebddb2c9adcf41c37c5de6f8f723b4936d2619892R43-R44), and I removed the rules for the hover state. In this way, we don't break the UI on TT2. I don't know if this is the best way. I'm open to suggestions :D 


Also, I noticed that with your suggestions, we fixed the `update cart` button when it is disabled 🥳 

#### Before
![image](https://user-images.githubusercontent.com/4463174/179989194-25d6ae22-42a9-47b0-80b1-0cc86e455535.png)

#### After

![image](https://user-images.githubusercontent.com/4463174/179989234-daac9631-03fe-4b04-92af-cf667ee79483.png)

Reply:
> Thanks for your suggestion! The selectors button.button, a.button with the hover state breaks the current UI with TT2

@gigitux I believe we're actually causing some regressions here at this point? 🤔 

Here is a side-by-side of this repo's `trunk` vs the latest from the `add/introduce-block-theme-default-styles` branch.

| TT2 `trunk` | TT2 `add/introduce-block-theme-default-styles` |
|---------|------------|
| ![woo-tt2-trunk](https://user-images.githubusercontent.com/405912/180308237-045196a5-a559-437e-91b2-ce092543bee8.gif) | ![woo-tt2-updates](https://user-images.githubusercontent.com/405912/180308418-588cc1ab-4c48-44a5-b2d1-12a6bfb74f6a.gif) |

Reply:
Are you referring to the gray button? It was a wanted fix:
> Also, I noticed that with your suggestions, we fixed the update cart button when it is disabled 🥳

The `update cart` button is disabled, so for me, it makes sense that it is gray.

Reply:
@gigitux you're right. I forgot about the disabled part and didn't tie the unique color treatment. 🤦 👍